tmail-backend/apps/distributed/pom.xml

```diff
@@ -19,6 +19,10 @@
             <artifactId>apache-mime4j-dom</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>blob-guice</artifactId>
+        </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>blobid-list</artifactId>
```

Comment:
Technically no need to inject anymore blobid-list here, that would be a transitive dep added via blob-guice?

Reply:
Nevermind, you likely still need it, but need to add the dep to secondary blobstore module too (cf error compile message on CI)

Reply:
Or maybe not... please double check